evol/evolution.py

```diff
@@ -160,15 +160,15 @@ def mutate(self, func, probability=1.0, name=None, **kwargs) -> 'Evolution':
         """
         return self._add_step(MutateStep(name=name, probability=probability, func=func, **kwargs))
 
-    def log(self, name=None, **kwargs) -> 'Evolution':
+    def log(self, name=None, every=1, **kwargs) -> 'Evolution':
```

Comment:
The `every` keyword is missing from the docstring.

Reply:
It is in there now.